uyuni-tools/last_update_export.py

```diff
@@ -0,0 +1,42 @@
+#! /usr/bin/python3
+
+import os
+import subprocess
+import datetime
+from xmlrpc.client import ServerProxy
+import ssl
+import socket
+
+SUMA_FQDN = socket.getfqdn()
+MANAGER_LOGIN = "admin"
+MANAGER_PASSWORD = "<change_me>"
+
+MANAGER_URL = "https://" + SUMA_FQDN + "/rpc/api"
+
+context = ssl.create_default_context()
+client = ServerProxy(MANAGER_URL, context=context)
+key = client.auth.login(MANAGER_LOGIN, MANAGER_PASSWORD)
+
+today = datetime.date.today()
+yesterday = today - datetime.timedelta(days=1)
+log_file_path = f"/mnt/logs/{today}-daily_export.log"
+output_dir = "/mnt/export/updates"
```

Comment:
The script is assuming these paths are existing but this is not usually the case.

Could you make these configurable as other parameters? Then it would be nice that the script will check first if the paths exist, to prevent problems.